api/Crypto.json

```diff
@@ -142,16 +140,10 @@
             "ie": {
               "version_added": false
             },
-            "nodejs": [
-              {
-                "version_added": "19.0.0"
-              },
-              {
-                "version_added": "16.7.0",
-                "version_removed": "19.0.0",
-                "notes": "From version 14.17.0, a similar method is available as <a href='https://nodejs.org/docs/latest-v14.x/api/crypto.html#crypto_crypto'><code>crypto.randomUUID()</code></a>."
-              }
-            ],
+            "nodejs": {
+              "version_added": "16.7.0",
+              "notes": "Since v15.6.0 and v14.17.0, a similar method is available via the <code>crypto</code> module."
```

Comment:
```suggestion
              "version_added": "16.7.0",
              "notes": "Since 14.17.0 and 15.6.0, a similar method is available via the <code>crypto</code> module."
```